src/main/java/seedu/lifeasier/parser/Parser.java

```diff
@@ -169,6 +172,34 @@ private Command parseShowNotesCommand(String input) {
         return new ShowNotesCommand(title);
     }
 
+    /**
+     * Parses the deleteNotes command that the user inputs.
+     * @param input String containing the user's input.
```

Comment:
Consider adding a line in between to ensure the JavaDoc standard is adhered to.
```suggestion
     * Parses the deleteNotes command that the user inputs.
     * 
     * @param input String containing the user's input.
```

Reply:
Same for a few of the other JavaDoc comments!